source/core/src/main/com/csse3200/game/areas/ForestGameArea.java

```diff
@@ -104,13 +116,106 @@ public void create() {
       player.getEvents().addListener("setPosition", this::handleNewChunks);
       player.getEvents().addListener("spawnKangaBoss", this::spawnKangarooBoss);
       player.getEvents().addListener("dropItems", this::spawnEntityNearPlayer);
+      player.getEvents().addListener("unlockArea", this::unlockArea);
       kangarooBossSpawned = false;
 
       //Initialise inventory and quests with loaded data
       player.getComponent(InventoryComponent.class).loadInventoryFromSave();
       player.getComponent(QuestManager.class).loadQuests();
   }
 
+  /**
+   * Unlock an area of the map
+   */
+  private void unlockArea(String area) {
```

Comment:
What is the point of this function (is this a part that hasn't been implemented yet)?